pkg/agent/manager/manager.go

```diff
@@ -94,7 +96,8 @@ type manager struct {
 
 	// backoff calculator for fetch interval, backing off if error is returned on
 	// fetch attempt
-	backoff backoff.BackOff
+	backoff         backoff.BackOff
```

Comment:
It may be confusing having a general `backoff` and then a specific `svidSyncBackoff`
Can we name the former something more specific?